packages/dbgpt-serve/src/dbgpt_serve/rag/service/service.py

```
@@ -591,6 +598,50 @@ async def async_doc_process(
+                        )
+                        chunks = await domain_index.transform(chunks)
+                        chunks = await domain_index.load(
+                            chunks,
+                            vector_store=storage_connector,
+                            max_chunks_once_load=max_chunks_once_load,
+                            max_threads=max_threads,
+                        )
+                        chunk_docs = chunks
+                        doc.chunk_size = len(chunk_docs)
+                        vector_ids = [chunk.chunk_id for chunk in chunk_docs]
+                    except Exception as domain_err:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)